### PR TITLE
Add keyboard icon

### PR DESCRIPTION
**Pull request by it-is-not**

### CODE REVIEW

File: tags.json [hunk 1]
```diff
@@ -1313,6 +1313,7 @@
     "authentication",
     "secure"
   ],
+  "keyboard": [],
```
Comment (ericfennis) [inline tags.json:1316]:
```suggestion
  "keyboard": [
    "layout"
    "spell",
    "settings",
    "mouse"
  ],
```

File: icons/keyboard.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect x="2" y="4" width="20" height="16" rx="2" ry="2" />
+  <path d="M6 8h.001m4 0h.001m4 0h.001m4 0h.001M8 12h.001m4 0h.001m4 0h.001M7 16h10" />
```
Comment (ericfennis) [inline icons/keyboard.svg:13]: Can you maybe split this up in separate parts, some design programs don't like this combined paths.
If you need any help with that let me know.

**Review by ericfennis — APPROVED** [review]
Nice work, I like the icon
Left some comments for the code.

File: tags.json (hunk 1 above)
Comment (it-is-not) [reply tags.json]: thanks, I couldn't think ideas for tags earlier

**Pull request merged**